### ISSUE TITLE
warnings from vue

### ISSUE THREAD

**Issue by olewaldo** [Luzifer/ots#158]
Hi,

I downloaded the latest [release](https://github.com/Luzifer/ots/releases/download/v1.11.0/ots_linux_amd64.tgz) and uploaded it to an ubuntu 22.04 server with plesk and nginx. I configured nginx as proxy:

```
location / {
	add_header X-Robots-Tag noindex;

	proxy_set_header Upgrade $http_upgrade;
	proxy_set_header Connection "Upgrade";
	proxy_set_header Host $host;
	proxy_set_header X-Real-IP $remote_addr;
	proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
	proxy_set_header X-Forwarded-Proto $scheme;

	proxy_pass http://127.0.0.1:3000/;
}
```

the ots file from the tgz archive runs as systemd service and redis. everything is working fine, but I get some warnings in my browser console:

vue-i18n] Fall back to translate the keypath 'text-powered-by' with 'de' locale. 
(and 9 more warnings like this one)

Download the Vue Devtools extension for a better development experience:
https://github.com/vuejs/vue-devtools
You are running Vue in development mode.
Make sure to turn on production mode when deploying for production.
See more tips at https://vuejs.org/guide/deployment.html

How can I fix this?

**Comment by Luzifer** [comment]
Very good catch! Thanks for reporting. The Vue Dev-Tools shouldn't be enabled in the release binaries. Fixed in 52a3474

**Comment by Luzifer** [comment]
Fix released as `v1.11.1` (will be available in the next few minutes)

**Comment by olewaldo** [comment]
Thank you very much for the super fast fix - v1.11.1 fixed the warnings :)
